fancy/analysis/results.py

```diff
@@ -43,7 +43,7 @@ def get_chain(self, list_of_keys):
 
         return chain
 
-    def get_truths(self, list_of_keys):
+    def get_truths(self, list_of_keys, composition=False):
```

Comment:
In the future we could think about a modular way to handle all the different analysis types, but low priority for the moment.